lib/utils/extensions.dart

```diff
@@ -61,6 +62,20 @@ extension JsonKeyModels on List<DartDeclaration> {
         .join('\n');
   }
 
+  String getEnums() {
+    return where((element) => element.isEnum)
+        .map((e) => e.getEnum().toTemplateString())
+        .where((element) => element != null && element.isNotEmpty)
+        .join('\n');;
```

Comment:
here have two semi colon
```suggestion
        .join('\n');
```